packages/snaps-execution-environments/src/offscreen/index.ts

```diff
@@ -0,0 +1,18 @@
+import { BrowserRuntimePostMessageStream } from '@metamask/post-message-stream';
+
+// import { executeLockdown } from '../common/lockdown/lockdown';
+// import { executeLockdownMore } from '../common/lockdown/lockdown-more';
+import { OffscreenSnapExecutor } from './OffscreenSnapExecutor';
+
+// TODO: Uncomment this.
+// executeLockdown();
+// executeLockdownMore();
```

Comment:
Somehow this suddenly broke. I need to look into this and uncomment before this can be merged.

Reply:
This should be fixed by bumping SES to `0.18.1`: https://github.com/endojs/endo/blob/master/packages/ses/CHANGELOG.md#0181-2022-12-23